app/Lib/lang.php

```diff
@@ -1670,6 +1672,10 @@
   'fd.ip' =>          'IP Address',
   'fd.job.complete_time' => 'Finished',
   'fd.job.finish_summary' => 'Finish Summary',
+  'fd.job.max_retry' => 'Maximum Retry',
+  'fd.job.max_retry.desc' => 'If the job fails, it will automatically be rescheduled for <bold>"%1$s" number</bold> of times. (To stop retrying, simply cancel this job.)',
```

Comment:
I think we could simplify this to "it will automatically be rescheduled up to %1$s time(s)".

Reply:
done